_posts/2019-01-23-multifile-python-functions.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Multifile python functions
+description: "Tidy python in an OpenFaaS world; or, what happens when you want to split your Python function into multiple modules?"
+date: 2019-01-23
+image: /images/multifile-python-function/pexels-book-stack-books-bookshop-264635.jpg
+categories:
+  - python
+author_staff_member: lucas
+dark_background: true
+---
+
+As a core contributor to OpenFaaS, I hangout in the OpenFaaS Slack a lot, you can [join us here][openfaas-slack-singnup]. Most recently that meant trying to answer this question
+
+> [How can I] write a function in python that uses multiple source files so that the main `handler.py` is able to import helper functions from separate source files.
+
+Today, I will walk though creating exactly this kind of function by creating a word count function.
+
+**TL;DR**: Use explicit relative imports to split your python code into several files. If you want to quickly jump to the finished code, I have create a [public repo here][project-repo].
```

Comment:
The post is so short that I don't think the TLDR is needed. 

python -> Python.